operations/validate.go

```diff
@@ -82,10 +82,24 @@ func validateFile(path string, ac *legacyClient, quiet bool) error {
 
 	if len(projErrors) != 0 {
 		grip.Info(projErrors)
-		return errors.Errorf("%s is an invalid configuration", path)
+	}
```

Comment:
Isn't the point of L79 to let you suppress warnings?

Reply:
This means that that the CLI will print warnings but nevertheless exit 0.